Docker/content-watcher.overview.md

```diff
@@ -0,0 +1,31 @@
+# Content Watcher Service Overview
+
+The Content Watcher Service is part of the [Gateway](https://projectlibertylabs.github.io/gateway) collection of services that provides a familiar callback API to retrieve content and publishing announcements from the [Frequency](https://docs.frequency.xyz/) chain. The service handles all of the necessary blockchain interaction and allows clients to interact using a familiar, Web2-friendly interface.
+
+## Key Features
+
+### API
+
+- **Parse DSNP Messages on Frequency**
+- **Send content to registered webhooks**
+- **Simple Schema and MSA Id based filtering**
+
```

Comment:
Note: Picky point, but it's better if these bullets are formatted the same.  So for the third bullet maybe:
"Filter via simple Schema and MSA Id-based filtering"